src/types.rs

```diff
@@ -558,6 +570,58 @@ impl<T: ToRedisArgs> ToRedisArgs for Option<T> {
     }
 }
 
+impl<T: ToRedisArgs + Hash + Eq> ToRedisArgs for HashSet<T> {
+    fn to_redis_args(&self) -> Vec<Vec<u8>> {
+        let mut rv : Vec<&T> = Vec::with_capacity(self.len());
+
+        rv.extend(self.iter());
+        ToRedisArgs::make_arg_vec_ref(rv.as_slice())
```

Comment:
`make_arg_vec_ref` was only added for this new use case and in both instances we double-allocate. We can simply inline this functionality for now